### PR TITLE
Initial commit to integrate conferences and applauses

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**

### CODE REVIEW

File: client/src/app/site/interaction/services/applause.service.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    presentUsers: number;
+}
+
+export enum ApplauseType {
+    particles = 'applause-type-particles',
+    bar = 'applause-type-bar'
+}
+
+const applausePath = '/system/applause'; // TODO
+const applauseNotifyMessageName = 'applause'; // TODO
```
Comment (GabrielInTheWorld) [inline client/src/app/site/interaction/services/applause.service.ts:21]: We need a new endpoint for the applause-calculation

**Comment by GabrielInTheWorld** [comment]
@tsiegleauq: Please have a look into this PR and give me a feedback. I have implemented the latest state of the interaction-module in OS3. To test this, you need the related backend-PR: OpenSlides/openslides-backend#829.

File: client/src/app/site/interaction/components/action-bar/action-bar.component.ts [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+@Component({
+    selector: 'os-action-bar',
+    templateUrl: './action-bar.component.html',
+    styleUrls: ['./action-bar.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    animations: [fadeInAnim, fadeInOutAnim]
+})
+export class ActionBarComponent extends BaseComponent {
+    public applauseLevel = 0;
+    public applauseDisabled = false;
```
Comment (tsiegleauq) [inline client/src/app/site/interaction/components/action-bar/action-bar.component.ts:26]: It appears you did not take the current OS3 for migrating this. These lines do not exist anymore.

https://github.com/OpenSlides/OpenSlides/blob/f1e2996c6ef880683a52b5bdc0548efa209fdd80/client/src/app/site/interaction/components/action-bar/action-bar.component.ts#L27-L30

**Review by tsiegleauq — APPROVED** [review]
It appears you were not using the current os3 master to port these features.
ensure to also cover these three PRs

https://github.com/OpenSlides/OpenSlides/pull/6103
https://github.com/OpenSlides/OpenSlides/pull/6129
https://github.com/OpenSlides/OpenSlides/pull/6121

**Comment by GabrielInTheWorld** [comment]
> It appears you were not using the current os3 master to port these features.
> ensure to also cover these three PRs
> 
> [OpenSlides/OpenSlides#6103](https://github.com/OpenSlides/OpenSlides/pull/6103)
> [OpenSlides/OpenSlides#6129](https://github.com/OpenSlides/OpenSlides/pull/6129)
> [OpenSlides/OpenSlides#6121](https://github.com/OpenSlides/OpenSlides/pull/6121)

Done.

**Review by tsiegleauq — APPROVED** [review]
Looks good so far.
Requires following backend issues to be resolved:
https://github.com/OpenSlides/openslides-backend/issues/813
https://github.com/OpenSlides/openslides-backend/issues/842

After this was done, rebase and generate a new package-lock.json (required for ci)

**Comment by GabrielInTheWorld** [comment]
I rebased, created a new `package-lock.json` and added the markers for the labels in the `meeting-settings-definition.ts`-file.

**Comment by tsiegleauq** [comment]
requires https://github.com/OpenSlides/openslides-backend/pull/829

**Comment by GabrielInTheWorld** [comment]
I will merge this, because OpenSlides/openslides-backend#829 was merged.

**Pull request merged**